seminar/index.html

```diff
@@ -29,145 +29,250 @@ <h6 class="logo">Creative techology seminar</h6>
     </header>
 
     <main>
-        <!-- WEEK 2 -->
-        <section>
-            <h2>What is technology and why?</h2>
-            <p class="date">Friday, Sept 24</p>
-            <p class="documentation">What do we call "technology"? What's it like? What does it want? How old is it? How do build relationships with it and through it? What should we ask ourselves before, during and after we engage with it?</p>
-            <ul>
-                <li><strong>Student Leads:</strong></li>
-                <li><strong>Reading:</strong> <a href="http://www.ursulakleguinarchive.com/Note-Technology.html" target="_blank">Ursula LeGuin, "A Rant About "Technology"</a>, <a href="https://theconvivialsociety.substack.com/p/the-questions-concerning-technology" target="_blank">L. M. Sacasas, “The Questions Concerning Technology”</a>, <a href="https://medium.com/backchannel/all-technology-is-assistive-ac9f7183c8cd" target="_blank">Sara Hendren "All Technology Is Assistive"</a></li>
-            </ul>
-        </section>
-        <hr>
-        <!-- WEEK 3 -->
-        <section>
-            <h2>Tools and making (tools)</h2>
-            <p class="date">Friday, Oct 01</p>
-            <p class="documentation"></p>
-            <ul>
-                <li><strong>Student Leads:</strong></li>
-                <li><strong>Reading:</strong> <a href="#" target="_blank"></a></li>
-            </ul>
-        </section>
-        <hr>
-        <!-- WEEK 4 -->
-        <section>
-            <h2>Imagination and design fiction</h2>
-            <p class="date">Friday, Oct 08</p>
-            <p class="documentation"></p>
-            <ul>
-                <li><strong>Student Leads:</strong></li>
-                <li><strong>Reading:</strong> <a href="Zaidi-Worldbuilding.pdf" target="_blank">Leah Zaidi "Worldbuilding in Science Fiction, Foresight and Design"</a></li>
-            </ul>
-        </section>
-        <hr>
-        <!-- WEEK 5 -->
-        <section>
-            <h2>AI and I</h2>
-            <p class="date">Friday, Oct 15</p>
-            <p class="documentation">Lorem ipsum dolor sit amet consectetur adipisicing elit. Debitis atque fugit illum dolor sunt, id reprehenderit assumenda ipsum corrupti perspiciatis, doloremque omnis deleniti necessitatibus eum, nihil facilis earum culpa laborum.</p>
+      <section class="seminar-title">
+        <h1>Creative<br />Technology<br />Seminar:</h1>
+      </section>
+      <!-- WEEK 2 -->
+      <section>
+        <h3>What is technology and why?</h3>
+        <p class="date">Friday, Sept 24</p>
+        <p class="project-brief">
```

Comment:
Can we make sure paragraphs occupy one line? Same for all .project-brief paragraphs